test/e2e/framework/profile_gatherer.go

```diff
@@ -25,6 +25,8 @@ import (
 	"strings"
 	"sync"
 	"time"
+
+	e2essh "k8s.io/kubernetes/test/e2e/framework/ssh"
```

Comment:
It looks like every import aliases the package name. Did we choose the wrong name?

Reply:
It was an explicit choice of the testing-commons group; both the name of the package and the alias; I actually put up the PR up without and was told to alias everything.

It is compatible with all the logging PRs that are going in where the package name was `log` but we are explicitly aliasing everything with `e2elog`.

Reply:
Not a fan but this isn't a hill I need to die on.